src/Microsoft.Framework.SecretManager/Program.cs

```diff
@@ -57,9 +58,13 @@ public int Main(string[] args)
             try
             {
                 var app = new CommandLineApplication();
+                app.FullName = "DNX UserSecrets";
                 app.Name = "user-secret";
                 app.Description = "Manages user secrets";
 
+                var assembly = typeof(Program).GetTypeInfo().Assembly;
+                app.ShortVersionGetter = () => assembly.GetName().Version.ToString();
```

Comment:
I am not sure if displaying the `Version` alone is enough...like this number doesn't change *within* a release where as the package versions keep changing...but since for every public release the package and assembly version match(correct?) this should probably be fine:

![image](https://cloud.githubusercontent.com/assets/6559784/8582964/f2fbbdf0-257e-11e5-8ea5-bf306d3b7375.png)

cc: @Tratcher